.github/workflows/ci.yaml

```diff
@@ -33,7 +41,7 @@ jobs:
         run: |
           python -m venv venv
           source venv/bin/activate
-          pip install -U neuro-cli packaging neuro-sdk
+          pip install -U neuro-cli neuro-sdk
```

Comment:
by the way, I imagine `neuro-cli` installs `neuro-sdk` as well.

Reply:
rmved in 24d9ab4